dcf/tests/test_notification_manger.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding:utf-8 -*-
```

Comment:
You should test new only functionality. No less, no more.
I expect your `TestCase` to cover:
- Message being sent for item which will be closed soon
- Item being deactivated if it was posted/updated more than `DCF_ITEM_MAX_AGE_DAYS` days before and message was sent to user
- Only one message was sent in each case